manager/controllers/app/m4dapplication_controller_test.go

```diff
@@ -388,11 +388,14 @@ var _ = Describe("M4DApplication Controller", func() {
 
 			// Create M4DApplication
 			Expect(k8sClient.Create(context.Background(), resource)).Should(Succeed())
+			time.Sleep(2 * time.Second)
```

Comment:
Should probably use Eventually instead of relying on sleep

Reply:
Done.